src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -2,7 +2,8 @@ DELETE FROM tab_calculation_method_rates WHERE calculation_method='BMI' AND form
 INSERT INTO tab_calculation_method_rates(calculation_method,formula_rate,formula,dimension) VALUES('DiseaseStates','PARAM_ChildPughScore','0','Dimensionless');
 INSERT INTO tab_calculation_method_rates(calculation_method,formula_rate,formula,dimension) VALUES('ExpressionParameters','TableFormulaWithXArgument_OntogenyFactor',NULL,'Dimensionless');
 INSERT INTO tab_calculation_method_rates(calculation_method,formula_rate,formula,dimension) VALUES('ExpressionParameters','TableFormulaWithXArgument_OntogenyFactorGI',NULL,'Dimensionless');
-UPDATE tab_calculation_method_rates SET formula='Age * 365.25/7 + GA' WHERE calculation_method='GFR_Calculation' AND formula_rate='PARAM_PostMenstrualAge';
+UPDATE tab_calculation_method_rates SET formula='((PMA*7/365.25)^Hill/(TM50^Hill+(PMA*7/365.25)^Hill)*(1-GFR_premat)+GFR_premat) * GFR_mat / V_kid_std * Renal_aging_scaling_factor' WHERE calculation_method='GFR_Calculation' AND formula_rate='PARAM_GFR_Spec';

```

Comment:
@Yuri05 This was the only place where this was used. The PMA in the formula is assumed to be in weeks. Well it's in year and we move it to weeks by "hand"

Reply:
it should be `PMA * 365.25 / 7`, not `PMA*7/365.25`

Reply:
Indeed